app/service/account.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import Depends
+from sqlmodel import Session
+from models.models import Account, AccountUpdate
+from service.auth import get_current_user
+from db import engine
+
+session = Session(bind = engine)
+
+def create_account(account:Account, token:str = Depends(get_current_user)) -> dict:
```

Comment:
```suggestion
from Typing import Dict 
def create_account(account:Account, token:str = Depends(get_current_user)) -> Dict:
```
p2; type에 대한 부분은 dict 보다 typing.Dict을 사용해주시면 좋을것 같습니다.

Reply:
네! 수정하였습니다!